src/main/java/com/gymrats/service/CalculatorService.java

```diff
@@ -0,0 +1,23 @@
+package com.gymrats.service;
+
+import com.gymrats.model.BmiNotLoggedRequest;
+import com.gymrats.model.BmiNotLoggedResponse;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CalculatorService {
+    float bmi;
```

Comment:
te zmienne nie są używane nigdzie poza jedną metodą nie widzę więc sensu wyciągania ich z metod, a nawet jeżeli to powinny być private